src/flatdirectory.js

```diff
@@ -79,15 +77,19 @@ export class FlatDirectory {
         const provider = new ethers.JsonRpcProvider(rpc);
         const fileContract = new ethers.Contract(address, FlatDirectoryAbi, provider);
         const [supportBlob, solcVersion] = await Promise.all([
```

Comment:
I think the `solcVersion` is a little confusing since it refers to solidity compiler version